### PR TITLE
remove shares

### PR DESCRIPTION
**Pull request by will-moore**
See https://github.com/ome/omero-web/issues/179

This removes the "Create Share" form, which is broken.
It is replaced with a message linking to https://www.openmicroscopy.org/omero/features/share/

Screenshot 2021-03-01 at 14 18 54

Also, it adds the same link under the Shares tab:

Screenshot 2021-03-01 at 14 33 57

### CODE REVIEW

**Comment by will-moore** [comment]
As discussed at stand-up this morning, it would be useful to have a similar "Not supported" warning under the "Shares" tab since the existence of that tab causes confusion.
The best URL to refer users to is still TBD.

**Comment by pwalczysko** [comment]
> As discussed at stand-up this morning, it would be useful to have a similar "Not supported" warning under the "Shares" tab since the existence of that tab causes confusion.
> The best URL to refer users to is still TBD.

For the new URL see suggestion in https://github.com/ome/www.openmicroscopy.org/pull/447

**Comment by will-moore** [comment]
Updated to point to new www.openmicroscopy.org/omero/features/share/ page from @pwalczysko PR: https://github.com/ome/www.openmicroscopy.org/pull/447

File: omeroweb/webclient/templates/webclient/base/includes/toolbar_forms.html [hunk 1]
```diff
@@ -3,7 +3,7 @@
 
 {% comment %}
 <!--
-  Copyright (C) 2011 University of Dundee & Open Microscopy Environment.
+  Copyright (C) 2021 University of Dundee & Open Microscopy Environment.
```
Comment (joshmoore) [inline omeroweb/webclient/templates/webclient/base/includes/toolbar_forms.html:6]:
```suggestion
  Copyright (C) 2011-2021 University of Dundee & Open Microscopy Environment.
```

**Review by joshmoore — COMMENTED** [review]
LGTM. (Minor patch suggested on the copyright)

**Comment by snoopycrimecop** [comment]
Conflicting PR. Removed from build [OMERO-python-superbuild-push#617](https://merge-ci.openmicroscopy.org/jenkins/job/OMERO-python-superbuild-push/617/). See the [console output](https://merge-ci.openmicroscopy.org/jenkins/job/OMERO-python-superbuild-push/617/consoleText) for more details.
Possible conflicts:
  - PR #149 will-moore 'Chown webclient'
    - omeroweb/webclient/views.py
  - PR #180 will-moore 'Update jquery'
    - omeroweb/webclient/views.py
  - PR #225 will-moore 'Query string ids'
    - omeroweb/webclient/views.py
  - PR #256 will-moore 'annotations api limit'
    - omeroweb/webclient/views.py
  - PR #264 will-moore 'omero table link to ROI'
    - omeroweb/webclient/views.py

--conflicts

**Comment by pwalczysko** [comment]
this PR is not built on merge-ci. Studying https://merge-ci.openmicroscopy.org/jenkins/job/OMERO-python-superbuild-push/618/console it seems that snoopy insists there is a conflict (from today early hours build)

**Comment by snoopycrimecop** [comment]
Conflicting PR. Removed from build [OMERO-python-superbuild-push#619](https://merge-ci.openmicroscopy.org/jenkins/job/OMERO-python-superbuild-push/619/). See the [console output](https://merge-ci.openmicroscopy.org/jenkins/job/OMERO-python-superbuild-push/619/consoleText) for more details.
Possible conflicts:
  - PR #180 will-moore 'Update jquery'
    - omeroweb/webclient/views.py
  - PR #225 will-moore 'Query string ids'
    - omeroweb/webclient/views.py

--conflicts

**Comment by pwalczysko** [comment]
Tested on merge-ci. Looks good.

**Review by pwalczysko — APPROVED** [review]

**Comment by jburel** [comment]
order for this PR
* Integrate this PR for next release
* Merge https://github.com/ome/www.openmicroscopy.org/pull/447 and publish it after release
* Merge https://github.com/ome/omero-guide-introduction/pull/19 and publish it after release

**Pull request merged**